iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/metadata/InformationSchemaTable.java

```diff
@@ -0,0 +1,79 @@
+package org.apache.iotdb.db.queryengine.plan.relational.metadata;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.iotdb.db.queryengine.common.header.ColumnHeaderConstant;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.Locale;
+import java.util.Optional;
+
+import static org.apache.iotdb.commons.schema.table.column.TsTableColumnCategory.MEASUREMENT;
+import static org.apache.iotdb.commons.schema.table.column.TsTableColumnCategory.TIME;
+import static org.apache.tsfile.read.common.type.FloatType.FLOAT;
+import static org.apache.tsfile.read.common.type.IntType.INT32;
+import static org.apache.tsfile.read.common.type.StringType.STRING;
+import static org.apache.tsfile.read.common.type.TimestampType.TIMESTAMP;
+
+public enum InformationSchemaTable {
```

Comment:
define it in `iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/schema/table`, because ConfigNode will also use it.